workers/event_catcher/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+gem "manageiq-messaging", "~> 1.0"
+gem "rbvmomi2", "~> 3.3"
+gem "sd_notify"
```

Comment:
Another thing we can do is use bundler inline and avoid the Gemfile.  As long as the GEM_HOME is accurate it should reuse the gemset, and then I don't think we need to care about being in the correct directory _nor_ using bundle exec.